Gujarati/Gujju

```diff
@@ -0,0 +1 @@
+Subproject commit 4cd29dd11a7ee8e8fdd74db312d222ea983884ef
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.